src/main/resources/config/application.properties-sample

```diff
@@ -42,7 +42,7 @@ webdriver.create.attempts = 3
 webdriver.proxy.enabled = false
 
 #optional. If path is not specified webdrivers will be searched in system PATH
-webdriver.drivers.path = src/test/resources/webdrivers/
+webdriver.drivers.path = src/test/resources/webdrivers/chromedriver.exe
```

Comment:
So now if we need to switch between browsers we should change two properties - webdriver.drivers.path and webdriver.browser.name, right?